myuw-ios/AppDelegate.swift

```diff
@@ -62,6 +63,35 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
         
         return true
     }
+    
+    // Handle deep links myuwapp://page
+    func application(_ application: UIApplication, open url: URL, options: [UIApplication.OpenURLOptionsKey : Any] = [:] ) -> Bool {
+        
+        print("app delegate deep link activated")
+        
+        // Determine who sent the URL.
+        let sendingAppID = options[.sourceApplication]
+        print("source application = \(sendingAppID ?? "Unknown")")
+        
+        if let scheme = url.scheme,
```

Comment:
Might move this logic into openDeepLink instead and do all the url parsing there.